solr/bin/solr.in.sh

```diff
@@ -229,4 +229,4 @@
 # Runtime properties are passed to the security policy file (server/etc/security.policy)
 # You can also tweak via standard JDK files such as ~/.java.policy, see https://s.apache.org/java8policy
 # This is experimental! It may not work at all with Hadoop/HDFS features.
-#SOLR_SECURITY_MANAGER_ENABLED=false
+SOLR_SECURITY_MANAGER_ENABLED=true
```

Comment:
We shouldn't put uncommented values in these files. After all, it says at the very top of the file:

> The default shipped state of this file is completely commented.

Instead, lets fix the actual defaults for the scripts. It just means if the environment variable is not set, we want to set it to `true`. Here is an example that does that for a different change, but demonstrates the idea: https://issues.apache.org/jira/secure/attachment/12989428/SOLR-14142.patch

Reply:
Thanks @rmuir for the information. I can roll back one commit to get back to there. I started with it, after reading the file and then uncommented this morning.

I think that maybe we can try to make this functionality more apparent to readers through a simple clause, which I will add to the PR for the top of file, in the same paragraph.

Reply:
The added comment is not wrong. Take a look at the patch I linked to again.

Reply:
got it. I think I understand.